doc/api/fs.md

```diff
@@ -775,6 +775,17 @@ including subdirectories and files.
 When copying a directory to another directory, globs are not supported and
 behavior is similar to `cp dir1/ dir2/`.
 
+### `fsPromises.exists(path)`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `path` {string|Buffer|URL}
+* Returns: {Promise} Resolved as `true` if the given path exists, otherwise,
+  resolved as `false`.
+
+Test whether or not the given path exists by checking with the file system.
```

Comment:
It doesn't really check if the file exists, which is why the name is misleading, see https://github.com/nodejs/node/issues/39960#issuecomment-910398093.

Reply:
I see. It makes sense. Do you have a better name suggestion? Or like [`os.path.exists`](https://docs.python.org/3/library/os.path.html#os.path.exists) on Python, is a sepecial note sufficient?